recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ test:
 about:
   home: https://github.com/OSGeo/proj-data
   license: Public domain | X/MIT | BSD-2/3/4-Clause | CC0 | CC-BY (v3.0 or later) | CC-BY-SA (v3.0 or later)
+  license_family: OTHER
```

Comment:
Sounds good.  I wasn't sure of "other" applied to multiple licenses, but since there isn't an option like "VARIOUS", this does seem closest.